Admin's Manual.md

```diff
@@ -130,3 +130,19 @@ There are 4 environment variables you can set to quickly disable integrations on
 * `DEVZAT_OFFLINE_SLACK=true` will disable Slack
 * `DEVZAT_OFFLINE_RPC=true` will disable the gRPC server
 * `DEVZAT_OFFLINE=true` will disable all integrations.
+
+### Making a private chat
+
+Devzat have been made to be a public chat-room. But if you want, you can use it as a private, invite-only chat-room. To do so, you should add the following lines in the configuration file:
+
+```yaml
+private: true # Enables the Whitelist checking
```

Comment:
I would just use the length of the whitelist to check if it's a private chat (len 0 == public, else private)

Reply:
This might not be a good idea. Something as restrictive as making the server private being behind an implicit settings feels dangerous.
Furthermore, as I followed your advice about allowing admins, a private server might have an empty whitelist.

Reply:
hmm. good points. I'm wondering if having it be implicit + having people add one admin id to the whitelist in the case that everyone allowed is an admin is a good system.

Reply:
I will do that if you really wants to, but I don't like having such a bug configuration switch be implicit.

Reply:
you're right, let's keep it explicit